pkg/apis/componentconfig/cloudcore/v1alpha1/types.go

```diff
@@ -124,6 +124,9 @@ type CloudHub struct {
 	// EdgeCertSigningDuration indicates the validity period of edge certificate
 	// default 365d
 	EdgeCertSigningDuration time.Duration `json:"edgeCertSigningDuration,omitempty"`
+	// TokenRefreshTime indicates frequence of Cloudcore token refresh
```

Comment:
plz declare the base unit of this configuration
```suggestion
	// TokenRefreshTime indicates frequence of Cloudcore token refresh, unit is hour.
```